css/common.styl

```diff
@@ -177,12 +179,21 @@ $owned-card
     display: flex
     flex-direction: column
     padding: 20px
```

Comment:
The collections card appears to have its own padding defined, so this `padding: 20px` is only applicable to the project card and should be moved into its style file.